js/apps/admin-ui/src/user/utils.ts

```diff
@@ -10,6 +14,14 @@ export const label = (attribute: UserProfileAttributeMetadata, t: TFunction) =>
     ? t(unWrap(attribute.displayName!))
     : attribute.displayName) || attribute.name;
 
+export const attributeGroupLabel = (
```

Comment:
Since this function is only used once, I think it would be preferential to co-locate it in the place it is used.